lint-rules-android-lint/src/test/java/com/vanniktech/lintrules/android/AnnotationOrderDetectorTest.kt

```diff
@@ -613,4 +637,18 @@ class AnnotationOrderDetectorTest {
         .run()
         .expectClean()
   }
+
+  @Test fun customNullableAnnotation() {
+    lint()
+        .allowCompilationErrors()
+        .files(java("""
+            package foo;
+
+            public class MyTest {
+              @Custom @Nullable public void myTest() { }
```

Comment:
Should not it be `@Nullable @Custom` here? Also to be consistent with singletonBeforeNonNullBeforeCustom

Reply:
In the SingletonBeforeNonNull case the problem that the Singleton is before nonnull. I was checking the fix there. 

Here this is correct. In the original logic, custom annotations are always forced to be the end. This was not valid before. Now they should be clean.